dts/bindings/gpio/nxp,pca95xx.yaml

```diff
@@ -16,6 +16,11 @@ properties:
       required: false
       description: Supports pull-up/pull-down
 
+    interrupt-gpios:
+      type: phandle-array
+      required: false
+      description: Interrupt GPIO pin
```

Comment:
The documentation should clarify that the signal is active-low (open drain) when produced by the hardware.

Reply:
fixed